library/standard_tables_capture.inc

```diff
@@ -327,7 +327,7 @@ function icd_import($type) {
     if( is_dir($dir) && $handle = opendir($dir)) {
         while (false !== ($filename = readdir($handle))) {
 	    // bypass unwanted entries
-	    if (!stripos($filename, ".txt") || stripos($filename,"diff")) {
+	    if (!stripos($filename, ".txt") || stripos($filename,"diff") || stripos($filename, "addenda")) {
```

Comment:
i upgraded from the 2016 set to 2017 by manualing adding the files to the contrib directory and issuing the raw sql update inside phpmyadmin, as you can see from the attached file, the addenda are not importing in the proper manner and are upsetting the selection of active=1 in the table too

[_icd10_dx_order_code___phpMyAdmin_4_5_4.pdf](https://github.com/openemr/openemr/files/431051/_icd10_dx_order_code___phpMyAdmin_4_5_4.pdf)